THEMES.md

```diff
@@ -78,10 +78,43 @@ https://user-images.githubusercontent.com/19476925/119483404-84223100-bd87-11eb-
 
 ## Ziro
 
-#### Blue Dark
+### Blue Dark
 
 ![Album](https://raw.githubusercontent.com/schnensch0/ziro/main/preview/album-blue-dark.png)
 
-#### Blue Light
+### Blue Light
 
 ![Artist](https://raw.githubusercontent.com/schnensch0/ziro/main/preview/artist-blue-light.png)
+
+### Gray Dark
+
+![Cards](https://raw.githubusercontent.com/schnensch0/ziro/main/preview/cards-gray-dark.png)
+
+### Gray Light
+
+![Search](https://raw.githubusercontent.com/schnensch0/ziro/main/preview/search-gray-light.png)
+
+### Green Dark
+![Podcast](https://raw.githubusercontent.com/schnensch0/ziro/main/preview/podcast-green-dark.png)
+
+### Green Light
+![Podcast](https://raw.githubusercontent.com/schnensch0/ziro/main/preview/podcast-green-light.png)
+
+### Orange Dark
+![Search](https://raw.githubusercontent.com/schnensch0/ziro/main/preview/search-orange-dark.png)
+
+### Orange Light
+![Library](https://raw.githubusercontent.com/schnensch0/ziro/main/preview/library-orange-light.png)
+
+### Purple Dark
+![Single](https://raw.githubusercontent.com/schnensch0/ziro/main/preview/single-purple-dark.png)
+
+### Purple Light
+![Playlist](https://raw.githubusercontent.com/schnensch0/ziro/main/preview/playlist-purple-light.png)
+
+### Red Dark
+![Profile](https://raw.githubusercontent.com/schnensch0/ziro/main/preview/profile-red-dark.png)
+
+### Red Light
+
+![Queue](https://raw.githubusercontent.com/schnensch0/ziro/main/preview/queue-red-light.png)
```

Comment:
These color schemes somehow follow the same pattern. Can you code an HTML table to make them more compact? Somethink like https://github.com/morpheusthewhite/spicetify-themes/tree/v2/Onepunch#dark

Reply:
would it be possible to do something like this:
#### Blue
| ![Album](https://raw.githubusercontent.com/schnensch0/ziro/main/preview/album-blue-dark.png) | ![Artist](https://raw.githubusercontent.com/schnensch0/ziro/main/preview/artist-blue-light.png) |
| :-: | :-: |
| Dark | Light |

#### Gray
| ![Cards](https://raw.githubusercontent.com/schnensch0/ziro/main/preview/cards-gray-dark.png) | ![Search](https://raw.githubusercontent.com/schnensch0/ziro/main/preview/search-gray-light.png) |
| :-: | :-: |
| Dark | Light |

like group them by color?

Reply:
I think it's fine